scripts/populate_mock_db.py

```
@@ -0,0 +1,75 @@
+            skills.append("IC")
+
+        crew_data.append((name, role, " ".join(skills), "42 hours"))
+
+    c.executemany(
+        "INSERT INTO crew (name, role, skills, contract_hours) VALUES (?, ?, ?, ?)",
+        crew_data,
+    )
+
+    # Add availability for some crew members
+    # Make 20 crew members available now
+    now = datetime.now()
```

Ruff (0.15.0)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)